### PR TITLE
02.12

### PR DESCRIPTION
**Pull request by pporc**
Hi there homework for 25.11 and 02.12

### CODE REVIEW

File: src/components/button/Button.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    super();
+    this.state = { btn: '' };
+  }
+
+  changeClass() {
+    if (this.state.btn === 'active') {
+      this.setState({ btn: '' });
+    } else {
+      this.setState({ btn: 'active' });
+    }
```
Comment (iliatcymbal) [inline src/components/button/Button.js:14]: ok, leave it as it is. But next time is better to use _btn_ as boolean flag _true/false_ and use `this.setState({ btn: !this.state.btn });`

File: src/components/button/Button.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  render() {
+    return (
+      <button className={this.state.btn} onClick={() => this.changeClass()}>
```
Comment (iliatcymbal) [inline src/components/button/Button.js]: why don't use just `onClick={this.changeClass}`?

**Review by iliatcymbal — APPROVED** [review]
good job

File: src/components/button/Button.js (hunk 2 above)
Comment (pporc) [reply src/components/button/Button.js]: @iliatcymbal changeClass was a class method, it did not work, fixed this

**Pull request merged**